pages/playlist/podcast/[id]/index.vue

```diff
@@ -1,3 +1,77 @@
+<script lang="ts" setup>
+const { t } = useI18n();
+toolbarTitleStore().setReactiveToolbarTitle(() => t("nav.podcast"));
+
+const { id } = useRoute<"playlist-podcast-id">().params;
+const collectionId = Number(id);
+
+const { data: podcast } = usePodcast({ id: collectionId });
+const { data: tracks, pending: tracksPending } = usePodcastTracks({
```

Comment:
Since podcasts are basically limitless, and the API applies some sort of limit here, we should at least indicate, that this might not be the full list, but a "load more" functionality is not yet implemented.